### ISSUE TITLE
[Accessibility: Error Identification] Device Ids input field not getting focused on validation failure

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/gingers390x#44]
In Add Adapter page, try to remove devices from black list without filling the device id field.
Error message displayed but input field is not getting focused.

![add san adapter](https://cloud.githubusercontent.com/assets/13199397/13976714/8b7c8a98-f0e8-11e5-881a-ff5d9fc61887.jpg)
